demos/image_processing_demo/cpp/README.md

```diff
@@ -0,0 +1,130 @@
+    -limit "<num>"            Optional. Number of frames to store in output. If 0 is set, all frames are stored.
+      -l "<absolute_path>"    Required for CPU custom layers. Absolute path to a shared library with the kernel implementations.
+          Or
+      -c "<absolute_path>"    Required for GPU custom kernels. Absolute path to the .xml file with the kernel descriptions.
+    -d "<device>"             Optional. Specify the target device to infer on (the list of available devices is shown below). Default value is CPU. Use "-d HETERO:<comma-separated_devices_list>" format to specify HETERO plugin. The demo will look for a suitable plugin for a specified device.
+    -pc                       Optional. Enables per-layer performance report.
+    -nireq "<integer>"        Optional. Number of infer requests. If this option is omitted, number of infer requests is determined automatically.
+    -nthreads "<integer>"     Optional. Number of threads.
+    -nstreams                 Optional. Number of streams to use for inference on the CPU or/and GPU in throughput mode (for HETERO and MULTI device cases use format <device1>:<nstreams1>,<device2>:<nstreams2> or just <nstreams>)
+    -loop                     Optional. Enable reading the input in a loop.
+    -no_show                  Optional. Do not show processed video.
+    -output_resolution        Optional. Specify the maximum output window resolution in (width x height) format. Example: 1280x720. Input frame size used by default.
+    -u                        Optional. List of monitors to show initially.
+```
+
+Running the application with the empty list of options yields an error message.
+
+To run the demo, you can use public or pre-trained models. To download the pre-trained models, use the OpenVINO [Model Downloader](../../../tools/downloader/README.md). The list of models supported by the demo is in [models.lst](./models.lst).
+
+> **NOTE**: Before running the demo with a trained model, make sure the model is converted to the Inference Engine format (\*.xml + \*.bin) using the [Model Optimizer tool](https://docs.openvinotoolkit.org/latest/_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html).
+
+You can use the following command to enhance the resolution of the images captured by a camera using a pre-trained single-image-super-resolution-1033 network:
+
+```
+./image_processing_demo -i 0 -m single-image-super-resolution-1033.xml -at sr
+```
+
+## Modes
+
+Demo application supports 3 modes:
+
+1. to display the result image.
+2. to display the original image with result together (left part is result, right part is original). Position of separator is specified be slider of trackbar.
+3. to display the difference image with result together. By analogy with second mode.
+
+User is able to change mode in run time using the next keys:
+
+* **R** to display the result.
+* **O** to display the original image with result.
+* **V** to display the difference image with result.
```

Comment:
Why not `D` key for the difference?

Reply:
The keys `C`, `M`, `D`, `H` are already used by monitors to represent information about resources (memory, processor).